cupy/linalg/_einsum.py

```diff
@@ -509,7 +509,7 @@ def einsum(*operands, **kwargs):
         optimize = 'greedy'
     if kwargs:
         raise TypeError('Did not understand the following kwargs: %s'
-                        % list(kwargs.keys))
+                        % list(kwargs))
```

Comment:
```suggestion
                        % list(kwargs.keys()))
```

Reply:
OK, changed it. Technically, `list(kwargs)` and `list(kwargs.keys())` will give the same result - a list of keys.

Reply:
Ah you're right 😄